### ISSUE TITLE
ImportError: No module named utils

### ISSUE THREAD

**Issue by pl-gauthier** [aspiers/ly2video#66]
Hi there,

Starting ly2video.py in the terminal gives me :
```
⇒  ly2video.py                                                                                                                                                                   -- INSERT --
Traceback (most recent call last):
  File "/usr/bin/ly2video.py", line 47, in <module>
    from utils import *
ImportError: No module named utils
```
On line 47 of the ly2video.py file we can find.
`from utils import *`

I am not aware of Python `utils` module and obviously, it is in your code.

I am installing it on Arch Linux from the AUR https://aur.archlinux.org/packages/ly2video-git

The problem is probably the way the PKGBUILD file is constructed from the package maintainer.
I thought I'd report anyway :-)

Good day!

**Comment by aspiers** [comment]
Yeah, obviously `ly2video.py` is not finding the provided `utils.py`.  I don't know anything about Arch Linux packaging, but actually [the `PKGBUILD`](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=ly2video-git) looks right to me.  It seems that the `setup.py` introduced by #51 is actually broken, because it is only including the Python modules under `ly`, not the ones at the top-level.  This should be very easy to fix.  I think we just need to create a new `ly2video/` directory and move the library files under that.

**Comment by aspiers** [comment]
This should now be fixed by d325d8a.
